install.sh

```diff
@@ -88,9 +88,11 @@ setup_ramalama() {
   $sudo install -m755 -d "$syspath"
   $sudo install -m755 "$to_file" "$ramalama_bin"
 
-  local python_files=("cli.py" "huggingface.py" "model.py" "ollama.py" "common.py" "__init__.py" \
-                      "quadlet.py" "kube.py" "oci.py" "version.py" "shortnames.py" "toml_parser.py")
-    
+  local python_files=("cli.py" "huggingface.py" "model.py" "ollama.py" \
```

Comment:
Wouldn't it be easier to just use a GLOB for this?

Reply:
The way this installer works is it pulls down install.sh via curl and then the various other files it needs via curl. We could maybe pull down the whole project as a zip/tarball , etc. and then do a glob, but we would pull a lot of unnecessary things also.

But yeah I've also been thinking maybe we can look into somehow making this smarter rather than listing all the files.